cassandra-diagnostics-commons/src/main/java/io/smartcat/cassandra/diagnostics/Query.java

```diff
@@ -4,69 +4,112 @@
  * This class represents a query report.
  */
 public class Query {
+
+    private long startTimeInMilliseconds;
+    private long executionTimeInMilliseconds;
+    private String executionErrorMessage;
+    private String clientAddress;
+    private String statementType;
+    private String keyspace;
+    private String tableName;
+    private String statement;
+
     /**
      * Query's execution start time.
+     * @return execution start timestamp
      */
-    public long startTimeInMilliseconds;
+    public long getStartTimeInMilliseconds() {
+        return startTimeInMilliseconds;
+    }
 
     /**
      * Query's execution time.
+     * @return execution time
      */
-    public long executionTimeInMilliseconds;
+    public long getExecutionTimeInMilliseconds() {
```

Comment:
I would skip using get in getters. It does not bring any meaning, and code reads much better when method is just `executionTimeInMilliseconds`.

@mgobec @nivancevic What do you think on this subject? This is as good time as any to bring this up and sync our code style.